python/python/tests/test_proto_coded.py

```diff
@@ -21,7 +21,7 @@ def test_encode_decode_delimited(self):
         b_arr = bytearray()
         ProtobufCodec.encode_delimited(b_arr, request)
         msg_len_varint = int(b_arr[0])
-        assert msg_len_varint == 18
+        assert msg_len_varint == 19
```

Comment:
Small question:
What is the context around this change?

Reply:
Very good question actually. Get/set commands used in this test got bigger numbers and now require more bytes to encode that.